flows/vision-zero/cris_import/cris_import.py

```diff
@@ -40,31 +35,71 @@
     insert_crash_change_template as insert_change_template,
 )
 
-kv_store = get_key_value("Vision Zero Development")
-kv_dictionary = json.loads(kv_store)
-
-SFTP_ENDPOINT = kv_dictionary["SFTP_ENDPOINT"]
-ZIP_PASSWORD = kv_dictionary["ZIP_PASSWORD"]
-VZ_ETL_LOCATION = kv_dictionary["VZ_ETL_LOCATION"]
-
-RAW_AIRFLOW_CONFIG_JSON = kv_dictionary["RAW_AIRFLOW_CONFIG"]
-RAW_AIRFLOW_CONFIG = json.loads(RAW_AIRFLOW_CONFIG_JSON)
-
-AWS_DEFAULT_REGION = kv_dictionary["AWS_DEFAULT_REGION"]
-AWS_ACCESS_KEY_ID = kv_dictionary["AWS_ACCESS_KEY_ID"]
-AWS_SECRET_ACCESS_KEY = kv_dictionary["AWS_SECRET_ACCESS_KEY"]
-AWS_CSV_ARCHIVE_BUCKET_NAME = kv_dictionary["AWS_CSV_ARCHIVE_BUCKET_NAME"]
-AWS_CSV_ARCHIVE_PATH_PRODUCTION = kv_dictionary["AWS_CSV_ARCHIVE_PATH_PRODUCTION"]
-AWS_CSV_ARCHIVE_PATH_STAGING = kv_dictionary["AWS_CSV_ARCHIVE_PATH_STAGING"]
-
-DB_HOST = kv_dictionary["DB_HOST"]
-DB_USER = kv_dictionary["DB_USER"]
-DB_PASS = kv_dictionary["DB_PASS"]
-DB_NAME = kv_dictionary["DB_NAME"]
-DB_IMPORT_SCHEMA = kv_dictionary["DB_IMPORT_SCHEMA"]
-
-DB_BASTION_HOST = kv_dictionary["DB_BASTION_HOST"]
-DB_RDS_HOST = kv_dictionary["DB_RDS_HOST"]
+SFTP_ENDPOINT = None
+ZIP_PASSWORD = None
+VZ_ETL_LOCATION = None
+
+AWS_DEFAULT_REGION = None
+AWS_ACCESS_KEY_ID = None
+AWS_SECRET_ACCESS_KEY = None
+AWS_CSV_ARCHIVE_BUCKET_NAME = None
+AWS_CSV_ARCHIVE_PATH_PRODUCTION = None
+AWS_CSV_ARCHIVE_PATH_STAGING = None
+
+DB_HOST = None
+DB_USER = None
+DB_PASS = None
+DB_NAME = None
+DB_SSL_REQUIREMENT = None
+
+DB_BASTION_HOST_SSH_USERNAME = None
+DB_BASTION_HOST = None
+DB_RDS_HOST = None
+
+if True:
```

Comment:
Yep, exactly. It feels easier to me when I can inspect the environment and configuration by typing `export` and even change it temporarily inside the dockerized agent instead of pushing up to prefect and pulling it back down at run-time. Doing this more elegantly is something I'm going to touch on in our airflow demo today, for sure.